File: src/main/java/de/tomjuri/examplemod/ExampleMod.java
package de.tomjuri.examplemod;

import cc.polyfrost.oneconfig.utils.commands.CommandManager;
import net.minecraftforge.fml.common.Mod;
import net.minecraftforge.fml.common.event.FMLInitializationEvent;

@Mod(modid = "examplemod", name = "Example Mod", version = "%%VERSION%%")
public class ExampleMod {

    private static ExampleMod instance;

    @Mod.EventHandler
    public void onInit(FMLInitializationEvent event) {
        new ExampleModConfig();
        instance = this;
        CommandManager.register(new ExampleCommand());
        System.out.println("Hello from ExampleMod!");
    }

    public static ExampleMod getInstance() {
        return instance;
    }
}

File: src/main/kotlin/de/tomjuri/examplemod/ExampleMod.kt
package de.tomjuri.examplemod

import cc.polyfrost.oneconfig.utils.commands.CommandManager
import net.minecraftforge.fml.common.Mod
import net.minecraftforge.fml.common.event.FMLInitializationEvent

@Mod(modid = "examplemod", name = "Example Mod", version = "%%VERSION%%")
class ExampleMod {

    companion object {
        lateinit var config: ExampleModConfig
    }

    @Mod.EventHandler
    fun onInit(event: FMLInitializationEvent) {
        config = ExampleModConfig()
        CommandManager.register(ExampleCommand())
        println("Sup world")
    }
}